src/query-jobs/boostr_impressions.js

```diff
@@ -6,7 +6,7 @@
 module.exports = async function main(config) {
   const query = `
     SELECT
-      timestamp_trunc(timestamp, day) AS Date,
+      FORMAT_TIMESTAMP("%m/%d/%Y", timestamp) as Date,
```

Comment:
Use `AS` for consistent formatting

Reply:
m/d/y is probably the most ambiguous date format available, so using it in a structured report seems risky

Reply:
Right?  I asked that same question (after they asked me to change it), but apparently this is the format they ingest.